designs/_posts/2016-09-30-repository-cache.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: contribute
+title: Central cache for external repositories
+---
+
+# Central cache for external repositories
+## Part 1: Caching HttpDownloader values
+
+**Author**: jingwen@google.com
+
+**Status**: Implemented
+
+**Reviewers**: kchodorow@, jcater@, dmarting@
+
+## Introduction
+
+With `bazel fetch //repo:...`, external repositories and artifacts are
+downloaded from remote sources with repository functions, such as
+`http_archive()` or `maven_jar()`. Sometimes, network access is not
+available, or redownloading of already-downloaded repositories is
+wasteful.
+
+With a central cache, Bazel can check it for the requested
+repositories before hitting the internet. This allows multiple
+workspaces to share the same repository cache. The cache should also
+be portable, i.e. tar’ing up the cache and reusing it on another
```

Comment:
nit: we favor "that is" instead of "i.e." (and "for example" instead of "e.g.")

Reply:
Done.